bootloader/test_infra/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "test_infra"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+vm = { git = "https://github.com/matter-labs/zksync-era.git", branch = "mmzk_1006_fix_tracer" }
```

Comment:
we'll migrate to `boojum-integration` in a subsequent PR, right?

Reply:
done.